bin/cli.js

```diff
@@ -129,6 +129,7 @@ import i18nTransform from '../dist/transform.js'
   }
 
   var count = 0
+  let warningMap = {}
```

Comment:
Since this is only used for duplicated keys I'd probably name this `duplicatedKeysCounts` or something similar.